src/astro/databases/base.py

```diff
@@ -183,13 +184,16 @@ def create_table_using_schema_autodetection(
         table: Table,
         file: Optional[File] = None,
         dataframe: Optional[pd.DataFrame] = None,
+        columns_names_capitalization: ColumnCapitalization = "lower",  # skipcq
```

Comment:
@utkarsharma2 another strategy here (I added this, so DeepSource does not complain for now) could be to raise an exception if this value is passed to the base database - since it is not being used. That said, this shouldn't be a blocker for the PR to be merged.